patches/node/fix_crypto_tests_to_run_with_bssl.patch

```diff
@@ -322,34 +314,25 @@ index 5f4fafdfffbf726b7cb39c472baa3df25c9794cf..d52376da2cddd90adcdf8a9b7dcd03e3
 +  library: /digital envelope routines|Cipher functions/,
  };
  
- const decryptError = common.hasOpenSSL3 ?
-@@ -222,7 +221,8 @@ function test_rsa(padding, encryptOaepHash, decryptOaepHash) {
+ const decryptError = hasOpenSSL3 ?
+@@ -223,7 +222,7 @@ function test_rsa(padding, encryptOaepHash, decryptOaepHash) {
    }, bufferToEncrypt);
  
  
 -  if (padding === constants.RSA_PKCS1_PADDING) {
-+  // BoringSSL does not support RSA_PKCS1_PADDING.
-+  if (false) {
++  if (!common.openSSLIsBoringSSL) {
```

Comment:
I like all these changes that check `common.openSSLIsBoringSSL` instead of just commenting out upstream code :+1: